recipe/meta.yaml

```diff
@@ -19,6 +23,7 @@ requirements:
     - setuptools
   run:
     - python
+    - certifi
 
 test:
```

Comment:
That seems reasonable. Though I think we need to add it to `files` under `test` so it moves to the testing directory.